service-templates/rust/plugin/src/errors.rs

```diff
@@ -0,0 +1,25 @@
+use crate::bindings::host::common::types::{Error, PluginId};
+
+#[derive(PartialEq, Eq, Hash)]
+pub enum ErrorType {
+    QueryResponseParseError,
+}
```

Comment:
Please use the `plugin_error!` macro to define errors.

Reply:
hm, must have found the one place to pull it from you didn't already replace. PR to handle the one isntance I found: #955